nexus_demos/config/system_bts/main.xml

```diff
@@ -3,6 +3,7 @@
     <ReactiveSequence>
       <IsPauseTriggered paused="{paused}"/>
       <PausableSequence pause="{paused}">
+        <AddMetadata />
```

Comment:
Into bikeshedding now but I wonder if there is a more explicative name for this node? It doesn't really add metadata, rather it is populating the metadata (if it exists, or creates a new one) with workcell task assignments.  Arguably something like `AddTaskAssignmentsToMetadata` is a mouthful but at least it is clear what is being done without having to read the source code

Reply:
oh yeah for sure, that's a much better name. I started with this capability to get something working as fast as possible, and didn't really bother to update the name. Now that things have cooled down a bit, I'll try to refactor as much as possible. Thanks for the callout